### PR TITLE
Fixed Windows compatibility

### PR DESCRIPTION
**Pull request by sonovice**
Fix for #130

### CODE REVIEW

File: src/bboxdevicecontext.cpp [hunk 1]
```diff
@@ -10,6 +10,7 @@
 //----------------------------------------------------------------------------
 
 #include <assert.h>
+#define _USE_MATH_DEFINES // needed by Windows for math constants like "M_PI"
```
Comment (lpugin) [inline src/bboxdevicecontext.cpp]: Can't we have this (and the following) in vrv.h within the #ifndef _WIN32 ?

File: src/midi/MidiMessage.cpp [hunk 2]
```diff
@@ -16,6 +16,10 @@
 #include <vector>
 #include <iostream>
 
+#ifdef _WIN32
+#include <iterator>
```
Comment (lpugin) [inline src/midi/MidiMessage.cpp]: This needs do to be modified in https://github.com/craigsapp/midifile . Maybe @craigsapp will make it without a pull request?

File: src/bboxdevicecontext.cpp (hunk 1 above)
Comment (sonovice) [inline src/bboxdevicecontext.cpp]: Depends on the include order. I will have a look...

Comment (sonovice) [inline src/bboxdevicecontext.cpp]: Nope, not working. The vrv.h is included after all the other header files.

File: src/midi/MidiMessage.cpp (hunk 2 above)
Comment (sonovice) [inline src/midi/MidiMessage.cpp]: Would it be okay to leave it here until @craigsapp modifies his repo? I would take care of removing it once the midifile lib has been changed.

**Comment by lpugin** [comment]
Could you try with this fix a27f9b85bffbe7ab858922566d7c6236d20295ea ?

**Comment by sonovice** [comment]
Works :-)

**Comment by lpugin** [comment]
Great. I merged it to develop. Could you check if it works after the merge before I delete the branch?

**Comment by sonovice** [comment]
Already did that, no problems at all.

**Pull request merged**